Note of installing Okteto in heterogeneous clusters (#216)
okteto/docs: src/content/self-hosted/aks.mdx

kubectl config view --minify | grep server
```

> > Our installation guides assume Okteto will be running in a dedicated cluster. If you want to install Okteto in a cluster with other workloads, we recomment to contact support for extra guidance.

Author: @rberrelleza what would be the right CTA here?

Reviewer: I think talk to us is the right CTA, but let's rephrase it. @bigash42 could your team help here?

> > Our installation guides assume Okteto will be running in a dedicated cluster. If you want to install Okteto in a cluster with other workloads, we recomment to contact support for extra guidance.

Reviewer: This is a thought . I don't think the "with other workloads." phrase is clear. 

```suggestion
> Our installation guides assume Okteto will be running in a dedicated cluster. We recommend [contacting our team](https://www.okteto.com/contact/)  if you plan on installing Okteto in a cluster with other workloads.
```

Reviewer 2: This reads well to me. /contact routes to sales and they will need to pass over to support, but I like sending them to that form so they can fill out with more context.
